### ISSUE TITLE
[Issue Report]: How to select active magic spell?

### ISSUE THREAD

**Issue by locksoft** [diasurgical/devilutionX#6982]
### Operating System

Nintendo Switch

### DevilutionX version

1.5.2

### Describe

I can open the spells window (it’s a bit hard as it involves + and B, and it requests two hands: it would be better to swap + and - functions). But then? I can’t find any key combination or press to choose one.

### To Reproduce

I can’t reproduce it.

**Comment by StephenCWills** [comment]
What you're describing doesn't appear to be a bug, but also I'm not really sure what you're talking about.

> it’s a bit hard as it involves + and B, and it requests two hands

Do you need to be able to play the game one-handed? That's not really something we were targeting with our control scheme.

Also, `B` opens up the speedbook, but you don't need the `+` button for anything in there. Is `+` supposed to represent the d-pad?

> it would be better to swap + and - functions

If `+` is the d-pad, then what is `-`? If these are supposed to be the `+` and `-` buttons, then what do they have to do with the speedbook?

> I can’t reproduce it.

What can't you reproduce? If your issue can't be reproduced, then how are we supposed to know what you're even talking about?

**Comment by StephenCWills** [comment]
Thinking a little bit harder about this, it seems that maybe you're talking about using the `+` button to bring up the spell hotkey menu. If so, then you can simply remap your controls using the `Padmapper` settings via the `Settings` option in the main menu. Given that it's already possible to remap controls, I'll be closing this issue. Let us know if I've completely misunderstood the issue.

**Comment by locksoft** [comment]
You’re right, Stephen. I got confused by the “terminology” and my lack of searching inside the settings. The “padmapper” options were the solution. Sorry. :)

On Switch the “select” button is “-“ (minus) and the “start” one is “+” (plus). Also on Nintendo consoles the “ok” and “cancel” button are reversed compared to other ones like Xbox and PlayStation. Unfortunately it’s not possible to remap the menu behavior in the settings, and this brings to curious situations: like when inserting the name for a new character, the B Button (down one) cancel the typing form.

If I can suggest a change for the Switch, maybe renaming “select” and “start” to “-“ and “+” and swapping “A” and “B” in the will help a lot

I also noticed that, touching the screen, a sort of touch control interface appears. I don’t think it may be useful on switch (it also moves some windows around), maybe an option to enable/disable it could be nice.

About “one handing playing”: I was referring to the fact it wasn’t possible to press “+” (what you call “start”) and one of the four buttons below without raising the left hand from the controller, because It can’t be done with the right hand. But I’ve remapped everything in a more usable way and the issue is solved. :)
